packages/cardano-services/docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: "3.9"
+version: '3.9'
```

Comment:
Lots of change on quotes style. We should look at adding some yml linter to make it consistent, maybe https://www.npmjs.com/package/eslint-plugin-yml ?

Reply:
Yes, sorry, my VSCode formats them consistently with the code (I suppose it is prettier).
Do you think is it better I revert all these changes?

Reply:
I don't mind either style so I approved the PR, but please create a ticket to setup tooling to prevent the inconsistencies in the future

Reply:
Yes, please revert the style change @iccicci. It's acceptable if we have linting in place, but done in a separate a comprehensive commit.

Reply:
14dfa2d68 should solve this